spec/requests/v1/vocabularies_spec.rb

```diff
@@ -124,15 +131,15 @@
     context 'when deleting vocabulary' do
       before do
         FactoryBot.create(:vocabulary)
-        delete '/api/v1/vocabularies/subjects'
+        delete '/api/v1/vocabularies/mythical_creatures'
```

Comment:
For this test, you may want to check for the presence of the mythical_creatures vocabulary before deleting it, just to prove that it actually existed before the delete succeeded.